src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayTests.cs

```diff
@@ -57,5 +57,11 @@ public void VirtualNetworkGatewayRouteApiTest()
         {
             NetworkResourcesController.NewInstance.RunPsTest("Test-VirtualNetworkGatewayBgpRouteApi");
         }
+
+        [Fact]
```

Comment:
@henry416 please make sure that this is a `CheckIn` test so it runs during the regular build